.github/workflows/package-data-analysis.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
           aws-region: ${{ env.AWS_REGION }}
           role-to-assume: ${{ secrets[matrix.GH_ACTIONS_ROLE_ARN_SECRET] }}
       - name: Package SAM app
-        uses: govuk-one-login/devplatform-upload-action@v3.8.1
+        uses: govuk-one-login/devplatform-upload-action@86306cee24f5578f26626dbc67bfd08910101ff6 # vskip-canary
```

Comment:
can we avoid using canary versions. you might be able to grab the actual value from another repo